webapp/components/test-file-results-table.js

```diff
@@ -335,15 +352,56 @@ class TestFileResultsTable extends WPTFlags(Pluralizer(AmendMetadataMixin(WPTCol
     return rows && rows.length && rows[0];
   }
 
+  computeTotals(rows) {
```

Comment:
Here is my question for this bit, and perhaps I should have asked a bit early. Now we have removed the `Harness status` tweak, it appears that the total displayed on the test-level[1] is consistent with the subtest-level [2]

[1]
![Screen Shot 2022-05-05 at 3 43 58 PM (2)](https://user-images.githubusercontent.com/8611520/167037039-1772b7f5-09d9-4780-9698-08886dd6a8e0.png)

[2]
![Screen Shot 2022-05-05 at 3 44 34 PM (2)](https://user-images.githubusercontent.com/8611520/167037060-7f836a71-d6f1-4b6d-b938-35f398d1f77e.png)

If that is the case, do we have to calculate them ourselves or simple pass it down when a user clicks on the test?

Reply:
This is a very good question. I did a more thorough investigation when you mentioned this. The problem arises with the differences in search query results.
The numbers that are displayed in views above the single test level are plucked from the search results array of the entire test suite:
https://github.com/web-platform-tests/wpt.fyi/blob/5cadbbc2df6ac3557f17e6487c0d5f1a5ddd49f2/webapp/views/wpt-results.js#L627-L636
This array is typically huge and representative of the entire number of tests in WPT (right now it's usually around 46,000).
When we view a single test, we separately query for the subtest results that exist in that single test path:
https://github.com/web-platform-tests/wpt.fyi/blob/5cadbbc2df6ac3557f17e6487c0d5f1a5ddd49f2/webapp/components/test-file-results.js#L103-L107
This individual test query doesn't supply this aggregate count of passes and totals.
So then, why don't we just pass down the passes and totals that exist in the test suite search query? The problem is that the results are in the form of this very large array, so we would have to traverse the array to find the correct individual test result data to get these totals. Due to this, I thought it might be better to just take the individual test query data and check the subtest array length in a slightly modified way.

It might be worthwhile to change this on the server side at some point to include a subtest totals count when querying info on a specific test.

Reply:
>> It might be worthwhile to change this on the server side at some point to include a subtest totals count when querying info on a specific test.

I agree with a fix on the server side. I do see a way to pass along test results without going through the very large array, similar to the way that `path` is being updated through a global variable, but messy. At the end of the day, your way is more accurate on a subtest level

The only thing I am worried is the potential discrepancy between the `total` on the test level and subtest level. We want to make sure that the data is consistent and has a single reliable source of truth. @foolip and I had the same discussions on https://github.com/web-platform-tests/wpt.fyi/issues/2825 but not in the scope of this PR